x-pack/plugins/cloud_security_posture/public/components/no_vulnerabilities_states.tsx

```diff
@@ -131,7 +130,7 @@ const IndexTimeout = () => (
           defaultMessage="Data should appear in less than 10 minutes after elastic-agent is successfully deployed. {docs}"
           values={{
             docs: (
-              <EuiLink href="https://ela.st/findings" target="_blank">
+              <EuiLink href="https://ela.st/Vulnerabilities" target="_blank">
```

Comment:
This link did not work for me, maybe it was changed by mistake?

Reply:
Yes it did. I'll correct that thanks

Reply:
This feature will be deleted -  https://github.com/elastic/kibana/pull/156779 .